scripts/commands/checkLinks.ts

```diff
@@ -95,6 +95,12 @@ const FILES_TO_IGNORES: { [id: string]: string[] } = {
   ],
 };
 
+// The files in the folders are not searched to see if their links
+// are valid.
+const FOLDERS_TO_IGNORES: string[] = [
+  "docs/api/qiskit/release-notes/*",
```

Comment:
This is outdated, right?

Reply:
This was added to avoid all the broken links we have in the release notes as well as all the links in FILES_TO_IGNORES. Those links are pending to be fixed.